src/App.css

```diff
@@ -1,38 +1,57 @@
 .App {
```

Comment:
in the future we want to bundle each component so it also has the css for it's parts inside